lib/storage/metadata/mongoclient/MongoClientInterface.js

```diff
@@ -354,8 +358,12 @@ class MongoClientInterface {
             },
         }, {
             updateOne: {
+                // eslint-disable-next-line
```

Comment:
This can be removed by using the following code instead
```js
                    '_id': objName,
                    'value.versionId': {
                        $gt: versionId,
                    },
```